templates/test/result.html.ep

```diff
@@ -23,7 +23,7 @@
 % end
 
 <div class="row">
-    <div class="col-sm-push-3 col-sm-9">
+    <div class="col-sm-9">
```

Comment:
this push thingy is a left over from a previous version of the page. Just remove it - a col-sm-9 is pointless on its own